v2/app/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.kapt)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.compose)
+    alias(libs.plugins.kotlin.serialization)
+    alias(libs.plugins.google.services)
+    alias(libs.plugins.google.hilt)
+}
+
+android {
+    namespace = "com.google.firebase.example.makeitso"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "com.google.firebase.example.makeitso"
+        minSdk = 26
```

Comment:
Curious: Why 26? Out of all Firebase products, the highest minSdk might be Firebase Auth which has `minSdk=23`. Does Make it so use any library with a higher minSdk?

Reply:
No, I used the recommended one when you create a project from AS. But it makes sense to match what we're doing in other Firebase repos. Updated to 23.